elixir/apps/web/lib/web/live/settings/identity_providers/microsoft_entra/show.ex

```diff
@@ -84,12 +84,16 @@ defmodule Web.Settings.IdentityProviders.MicrosoftEntra.Show do
         </.button>
       </:action>
       <:help>
-        Directory sync is enabled for this provider. Users and groups will be synced every 10
-        minutes on average, but could take longer for very large organizations.
-        <.website_link path="/kb/authenticate/directory-sync">
-          Read more
-        </.website_link>
-        about directory sync.
+        <p>
+          Directory sync is enabled for this provider. Users and groups will be synced every 10
```

Comment:
I noticed you changed some of the other provider messages from `every 10 minutes` to `every few minutes`, were some of them left at 10 min intentionally?

Reply:
good catch, updated the rest